RELEASING.md

```diff
@@ -0,0 +1,19 @@
+Release Process
+===============
+
+ 1.  Ensure `VERSION_NAME` in `gradle.properties` is set to the version you want to release.
+ 2.  Add an entry in `CHANGELOG.md` with the changes for the release.
+ 3.  Update `README.md` with the version about to be released. Also update the RxJava version in
+     this file to its latest.
+ 4.  Update the RxJava version in `rxandroid/buld.gradle` to its latest. (We tell people that we
+     won't be tracking RxJava releases, and we don't, but we do it anyway when we are releasing for
+     those who ignore the advice.)
+ 5.  Commit: `git commit -am "Prepare version X.Y.X"`
+ 6.  Tag: `git tag -a X.Y.Z -m "Version X.Y.Z"`
```

Comment:
Suggestion: Replace with `git tag -am "Version X.Y.Z" X.Y.Z`
Reason: Combine single character options (consistent with command in the line above), and move tag name to the end (It will work either way, but more consistent with Git documentation).